src/protobuf-net.Grpc.AspNetCore/ServicesExtensions.cs

```diff
@@ -43,7 +43,7 @@ public CodeFirstServiceMethodProvider(ILoggerFactory loggerFactory, BinderConfig
             void IServiceMethodProvider<TService>.OnServiceMethodDiscovery(ServiceMethodProviderContext<TService> context)
             {
                 int count = new Binder(_logger).Bind<TService>(context, _binderConfiguration);
-                if (count != 0) _logger.Log(LogLevel.Information, "RPC services being provided by {0}: {1}", typeof(TService), count);
+                if (count != 0) _logger.Log(LogLevel.Information, "RPC services being provided by {service}: {count}", typeof(TService), count);
```

Comment:
Hi
The convention is to have PascalCase:
[CA1727: Use PascalCase for named placeholders](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1727)
```suggestion
                if (count != 0) _logger.Log(LogLevel.Information, "RPC services being provided by {Service}: {Count}", typeof(TService), count);
```